src/pyloidal/cocos.py

```diff
@@ -35,15 +34,14 @@ class Sigma:
 
     def __post_init__(self):
         if self.B_poloidal not in (-1, 1):
-            raise ValueError(
-                f"B_poloidal should be either 1 or -1, found {self.B_poloidal}"
-            )
+            msg = f"B_poloidal should be either 1 or -1, found {self.B_poloidal}"
+            raise ValueError(msg)
```

Comment:
I don't care for the warning that recommends using this `msg` intermediate variable, I don't think the improves anything -- but it's fine to keep now it's changed

Reply:
Yeah, I have mixed feelings. I agree with the premise that it leads to slightly more readable tracebacks, but it does so at the cost of slightly less readable code. (It's also a little annoying that it caused me to fail on code coverage)